assets/js/modules/adsense/components/setup/v2/SetupAccountSiteGettingReady.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * AdSense Setup Account Site Getting Ready component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useContext } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import ViewContextContext from '../../../../../components/Root/ViewContextContext';
+import { trackEvent } from '../../../../../util';
+import { MODULES_ADSENSE } from '../../../datastore/constants';
+import SetupAccountSiteUI from './common/SetupAccountSiteUI';
+const { useSelect } = Data;
+
+export default function SetupAccountSiteGettingReady() {
```

Comment:
Maybe let's create a new `SetupAccountSite` folder to contain `GettingReady`, `NeedsAttention`, `Ready`, and `RequiresReview` components? In this case, we will have more concise names.

Reply:
Good idea.